config/releases.exs

```diff
@@ -15,10 +15,6 @@ config :api_accounts,
   table_prefix: System.get_env("DYNAMO_TABLE_PREFIX"),
   migrate_on_start: true
 
-config :api_web, ApiWeb.Endpoint, secret_key_base: System.get_env("SECRET_KEY_BASE")
-
-config :api_web, :signing_salt, System.get_env("SIGNING_SALT")
-
```

Comment:
suggestion: You might want to keep these but make them optional. Otherwise you might have a hard time getting the ElasticBeanstalk deploy to take the first time during the transition, and when the API migrates to ECS these will go back to being loaded by environment variables anyways.